src/host/main.js

```diff
@@ -78,41 +94,42 @@ export async function hostMain({
       });
 
       const file = plan.files.find(({ name }) => name === test.filepath);
-      const result = await agent.run(JSON.parse(textDecoder.decode(file.bufferData)));
-      if (callbackUrl) {
-        const headers = {
-          'Content-Type': 'application/json',
-          ...(callbackHeader || {}),
-        };
-        const { testId, presentationNumber, capabilities, commands } = result;
-        const body = JSON.stringify({
-          testCsvRow: testId,
-          presentationNumber,
+      const testSource = JSON.parse(textDecoder.decode(file.bufferData));
+
+      const callbackBody = {
+        testCsvRow: testSource.info?.testId,
+        presentationNumber: testSource.info?.presentationNumber,
+      };
+
+      try {
+        postCallback({ ...callbackBody, status: 'TEST_STARTED' });
```

Comment:
Since we're not awaiting the "test started" message, this represents a race condition: ARIA-AT App could potentially receive the requests out-of-order. It may be appropriate to reconsider the `callbackRequests` abstraction in this patch.

Reply:
*nods* this was something I was thinking about as well